chapters/packages.tex

```diff
@@ -326,3 +326,114 @@ \section{External resources}\label{external-resources}
 \filename{modelica://Modelica/Mechanics/C.jpg} is illegal.
 The references \filename{modelica://Modelica.Mechanics/C.jpg} and \filename{modelica://Modelica/C.jpg} must also refer to two distinct resources.
 \end{example}
+
+\section{Multilingual Descriptions}\label{multilingual-descriptions}
+
+\begin{nonnormative}
+Descriptive texts in a model or library are usually formulated in English.
+To support users who are not so familiar with this, a Modelica tool can support the translation of these texts into any language.
+This should be done using external files, so that no modification of the actual Modelica text is required.
+\end{nonnormative}
+
+The files to support translation must be provided by the developer of the library.
+They must be stored in a subdirectory of the \filename{Resources} directory of the library with the name \filename{Language}.
+
+Two kind of files have to be provided:
+\begin{enumerate}
+\item template file \filename{<LibraryName>.pot}.
+It contains all necessary information to translate all descriptions, but no translations.
+The pattern \filename{<LibraryName>} denotes the toplevel class name of the library.
+\item one file for each supported language with the name \filename{<LibraryName>.<language>.po}.
+This file is a copy of the associated template file, but extended with the translations in the specified language.
+The pattern \filename{<language>} stands for the ISO 639-1 language code, e.g. de or en.
+\end{enumerate}
+
+The files consist of a header and a body.
+All text strings are in double quotes and encoded with UTF-8 characters.
+Comments start with an \lstinline!#! and are continued until the end of line.
+Spaces outside strings are ignored and used as separators.
+The detailed format of these files is described in \href{https://www.gnu.org/software/gettext/manual/gettext.pdf}{GNU gettext}.
```

Comment:
Isn't it dangerous to repeat the specification of the file format, rather than relying on the external gettext specification?  Maybe it would be better to just have a non-normative section with some examples of how the format could be used?

Reply:
Yes, maybe its better to have it as non-normative text.

Reply:
I added that in https://github.com/modelica/ModelicaSpecification/pull/3131

Reply:
I see that it at least made some parts non-normative.  Resolving and making another comment below.